app/views/lightweight_activities/edit.html.haml

```diff
@@ -26,6 +26,10 @@
       .field
         = f.label :theme_id, 'Theme'
         = f.select :theme_id, options_from_collection_for_select(Theme.all, 'id', 'name', @activity.theme_id), { :include_blank => "None (inherit from sequence, project, or site default)" }
```

Comment:
It would be best to disable the theme setting if the runtime is AP. As far as I know the theme will have no effect in the AP, so disabling this will make it more clear.